src/client_state.cpp

```diff
@@ -582,9 +582,13 @@ int wsrep::client_state::begin_nbo_phase_one(
 
     int ret;
     bool timed_out;
+
+    toi_meta_ = wsrep::ws_meta();
```

Comment:
Just for my understanding: I agree that the flow is more clear now, but before the result of this method was the same, no?

Reply:
Yes. It is the same, but the current flow ensures that we clear it in the beginning as well. Currently these get cleared only in `wsrep::client_state::leave_toi_common()` for TOI and [here](https://github.com/percona/wsrep-lib/blob/7d1e0517234b72b174a853b537f14be808e6b7e4/src/client_state.cpp#L737-LL739) for NBO.